app/services/key_rotator/hmac_fingerprinter.rb

```diff
@@ -14,7 +14,7 @@ def rotate(user:, pii_attributes: nil, profile: nil)
 
     def rotate_email_fingerprint(user)
       ee = EncryptedAttribute.new_from_decrypted(user.email)
-      user.update_columns(email_fingerprint: ee.fingerprint)
+      UpdateUser.new(user: user, attributes: { email_fingerprint: ee.fingerprint }).call
```

Comment:
is it important for this to be [`update_columns`](http://apidock.com/rails/ActiveRecord/Persistence/update_columns) instead of `update!` ? 

Behavior is slightly different but changing this breaks no tests. If we want to revert, I can create a ticket to write tests for this so we know not to change it.

cc @monfresh @pkarman